Source/Core/VideoCommon/VideoBackendBase.cpp

```diff
@@ -255,6 +248,15 @@ const std::vector<std::unique_ptr<VideoBackendBase>>& VideoBackendBase::GetAvail
     backends.push_back(std::make_unique<Vulkan::VideoBackend>());
 #endif
 #endif
+
+#ifdef _WIN32
+    backends.push_back(std::make_unique<DX11::VideoBackend>());
+    backends.push_back(std::make_unique<DX12::VideoBackend>());
+#endif
+#ifdef HAS_OPENGL
+    backends.push_back(std::make_unique<OGL::VideoBackend>());
+#endif
+
```

Comment:
Did you move this to the back intentionally? This makes Vulkan the implicit default on Linux, rather than OpenGL. (And it's the reason why the OpenGL FifoCI builder suddenly started running Vulkan.)

Reply:
Ah, that was for testing purposes since I couldn't set the video in nogui. Good catch, I'll revert it

Reply:
OatmealDome did, I just added the comment in here ;)

Reply:
Thanks @OatmealDome :1st_place_medal: 

Went ahead and reverted that